webui-src/app/theme.css

```diff
@@ -305,12 +305,20 @@ Target specific sections
 
 /* Navbar */
 nav.tab-menu {
+  border-right: 5px solid #3ba4d7;
   background-color: #333;
-  position: fixed;
-  top: 0;
-  width: 100%;
   border-bottom: 5px solid #3ba4d7;
   box-shadow: 0 5px 5px #aaa;
+  list-style: none;
```

Comment:
`list-style` only affects elements which have `display:list-item` so it's not used here.